### ISSUE TITLE
[fluorine] opensuse pkg tests failing

### ISSUE THREAD

**Issue by Ch3LL** [saltstack/salt-jenkins#1222]
The following pkg tests are failing on opensuse:

```
integration.states.test_pkg.PkgTest.test_pkg_cap_001_installed
integration.states.test_pkg.PkgTest.test_pkg_cap_002_already_installed
integration.states.test_pkg.PkgTest.test_pkg_cap_004_latest
integration.states.test_pkg.PkgTest.test_pkg_cap_005_downloaded
integration.states.test_pkg.PkgTest.test_pkg_cap_006_uptodate
integration.states.test_pkg.PkgTest.test_pkg_cap_003_installed_multipkg_with_version
```

https://jenkinsci.saltstack.com/job/fluorine/job/salt-opensuse-py2/47/

**Comment by terminalmage** [comment]
Can reproduce this by running `integration.states.test_pkg` as a whole. It looks like one or more of the tests don't clean up the package properly, and this causes a cascade of other failures as they are designed to run with the targeted package not present. I've set up try/finally blocks in each of the pkg_cap tests to ensure that the package is cleaned up, and I'm spawning a new box to run the tests.

**Comment by terminalmage** [comment]
Fixed via #50771.

**Comment by Ch3LL** [comment]
tests are passing
